index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my portfolio</title>
+    <link rel="stylesheet" href="styling.css">
+  </head>
+  <body>
+    <div class="container">
+      <section class="section1">
+        <header>
+          <nav class="navigation">
+            <section class="logo">
+              <a href="index.html" class="logo-text touch rough">my logo</a>
+            </section>
+            <section class="item-nav">
+              <a href="menu-items" class="menu-bar touch rough"><img src="mobile tag.png" alt="menu bar"></a>
+              <p class="menu-items">
+                <section class="items touch rough">
+                  <p class="item1">portfolio</p>
+                  <p class="item2">about</p>
+                  <p class="item3">contact</p>
+                </section>
+              </p>
+            </section>
+
+            <section class="mail-button items">
+              <p class="mail touch rough"><img src="Union.png" alt="mail link"></p>
+            </section>
+          </nav>
+        </header>
+      </section>
+      <section class="section2">
+        <article class="intro-text">
+          <p class="text1">Hey There.<br>I'm Chris.</p>
+          <p class="text2">I'm a Software Developer</p>
+          <p class="text3">I can help you build a product , feature <br> or
+            website. Look through some of my <br> work and experience! If you
+            like what you <br> see and have a project you need coded,<br>
+            don’t hesitate to contact me.</p>
+        </article>
+        <article class="icons touch rough">
+          <p class="medium"><img src="medium.svg" alt="medium site"
+              class="icon-sizing"></p>
+          <p class="github"><img src="github.svg" alt="github site"
+              class="icon-sizing"></p>
+          <p class="angellist"><img src="angel list.svg" alt="angellist site"
+              class="icon-sizing"></p>
+          <p class="linkedn"><img src="linkedn.png" alt="linkedn site"
+              class="icon-sizing"></p>
+          <p class="twitter"><img src="twitter.png" alt="twitter site"
```

Comment:
- [ ] I see that you used the semantic elements wrongly, so I instead of writing replace this with that I make to to you like a diagram to understand it and convert it into a code to better understand the structure of the page 😃

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Diagram design| Another diagram|
| --- | --- |
|   ![image](https://user-images.githubusercontent.com/83360636/227523970-5134d3e8-5672-46ba-b1f3-18686fab1405.png)   | ![image](https://user-images.githubusercontent.com/83360636/227525106-52271a7b-8c91-4851-968a-dc9fea19345b.png)|